### ISSUE TITLE
[Flexi] Output led for manually mapped buttons

### ISSUE THREAD

**Issue by centomila** [git-moss/DrivenByMoss#366]
Hi Moss!

I have a Vestax VCM-600, and I'm attempting to obtain LED feedback on manually mapped controls.

![image](https://medias.audiofanzine.com/images/normal/vestax-vcm-600-30698.jpg)

All the buttons are sent as MIDI Notes (Not CC).

Send
![image](https://github.com/git-moss/DrivenByMoss/assets/26800547/195e97a2-8ac2-4c6a-8d05-ec41749f4abc)

E.g.: HI Mute BUTTON 1= C3 Channel 1

Receive
![image](https://github.com/git-moss/DrivenByMoss/assets/26800547/a342d3c9-1828-4ac0-bfc6-049645e02b31)

When I map a function and select:
- "Send Value To Device: ON"
- "Send value to device when received (only buttons): ON"

Everything works as expected. The LEDs turn ON/OFF, and everything is smooth 🥳 .

However, I encounter a problem with Bitwig's MIDI mapping. Although I can map and use it, I don't get any LED feedback.

Here's what I've tried:
Test 1:
- Create a slot without function associated

Test 2:
- Assign a function (e.g., "Arm Rec") and then remap it.
  - The input is directed to the remapped button, but the output remains from the original mapping ("Arm Rec").

Test: 3
- Changed the Knob Mode
  
Test 4
- Set "Keyboard MIDI Channel" to OFF and to ALL.
  - This change doesn't seem to affect the output.
  
Test 5
- Switched the input remap method from "Controller Script" to "RAW MIDI".
  - This also didn't have any effect.

In short, what i think i need is a generic "Send value to device when received" for unassigned buttons.

I don't know if i'm doing something wrong or it's not possible at all. What i'm missing? 😩

Thanks for your project and support 🙂

**Comment by centomila** [comment]
> Bitwigs MIDI mapping does not send any data to the device.
> 

Oh no! I was sure this was possible. Thanks anyway.


> EDIT: Since you have now project parameters in Bitwig 5 there is no real need to use the Bitwig mapping.
> 

I've tried the project remote road and the track remotes road but i can't achieve exactly what i'm looking for because I need more than 8 macros.

Do you plan (if is possible with the current API) to add a way to map non-active pages in the project remotes?

I mean something like this in FLEXI

Project Remotes:
```
Project Remotes: Set Parameter 1 - Page 1
Project Remotes: Set Parameter 2 - Page 1
Project Remotes: Set Parameter 3 - Page 1
Project Remotes: Set Parameter 4 - Page 1
Project Remotes: Set Parameter 5 - Page 1
Project Remotes: Set Parameter 6 - Page 1
Project Remotes: Set Parameter 7 - Page 1
Project Remotes: Set Parameter 8 - Page 1

Project Remotes: Set Parameter 1 - Page 2
Project Remotes: Set Parameter 2 - Page 2
Project Remotes: Set Parameter 3 - Page 2
Project Remotes: Set Parameter 4 - Page 2
Project Remotes: Set Parameter 5 - Page 2
Project Remotes: Set Parameter 6 - Page 2
Project Remotes: Set Parameter 7 - Page 2
Project Remotes: Set Parameter 8 - Page 2

Project Remotes: Set Parameter 1 - Page 3
Project Remotes: Set Parameter 2 - Page 3
Project Remotes: Set Parameter 3 - Page 3
Project Remotes: Set Parameter 4 - Page 3
Project Remotes: Set Parameter 5 - Page 3
Project Remotes: Set Parameter 6 - Page 3
Project Remotes: Set Parameter 7 - Page 3
Project Remotes: Set Parameter 8 - Page 3

[...To page 8]
```

**Comment by git-moss** [comment]
No, I have no plans for this.
